pkg/services/team/teamimpl/store.go

```diff
@@ -336,6 +337,27 @@ func (ss *xormStore) GetByUser(ctx context.Context, query *team.GetTeamsByUserQu
 	return queryResult, nil
 }
 
+// GetIDsByUser is used by the Guardian when checking a users' permissions
+func (ss *xormStore) GetIDsByUser(ctx context.Context, query *team.GetTeamIDsByUserIDQuery) ([]int64, error) {
+	queryResult := make([]int64, 0)
+	err := ss.db.WithDbSession(ctx, func(sess *db.Session) error {
+		var params []any
+		params = append(params, query.UserID, query.OrgID, query.OrgID)
+		return sess.SQL(`SELECT t.id
+FROM team as t
+INNER JOIN team_member as tm ON t.id = tm.team_id
+WHERE
+tm.user_id=?
+AND tm.org_id=?
```

Comment:
Random thought: weird that the team membership table even has org IDs, but oh well...